File: fingerprint-with-qr-scan-flutter/fingerprint_with_qr_scan_flutter/lib/models/employee.dart
class Employee {
  int EmployeeId;
  String EmployeeCode;
  String FullName;
  String Email;
  String Phone;
  String DeviceId;

  Employee({
    required this.EmployeeId,
    required this.EmployeeCode,
    required this.FullName,
    required this.Email,
    required this.Phone,
    required this.DeviceId,
  });

  // Factory constructor to create an Employee object from JSON
  factory Employee.fromJson(Map<String, dynamic> json) {
    return Employee(
      EmployeeId: json['employeeId'] as int,
      EmployeeCode: json['employeeCode'] as String,
      FullName: json['fullName'] as String,
      Email: json['email'] as String,
      Phone: json['phone'] as String,
      DeviceId: json['deviceId'] as String,
    );
  }

  // Method to convert Employee object to JSON
  Map<String, dynamic> toJson() {
    return {
      'employeeId': EmployeeId,
      'employeeCode': EmployeeCode,
      'fullName': FullName,
      'email': Email,
      'phone': Phone,
      'deviceId': DeviceId,
    };
  }
}

File: biometric_waste_managment_solution/bwms_core_domain/ResidentsModels/Employee.cs
using bwms_core_domain.SystemModels;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace bwms_core_domain.UserModels
{
    public class Employee : User
    {
        public int EmployeeId { get; set; }
        public string FullName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string EmployeeCode { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public string DeviceId { get; set; } = string.Empty;
    }
}
